src/parallel/par.rs

```diff
@@ -174,12 +188,49 @@ macro_rules! par_iter_view_wrapper {
         }
     }
 
+    impl<'a, A> Producer for ParallelProducer<$view_name<'a, A, Ix1>>
+        where A: $($thread_bounds)*,
+    {
+        type Item = <$view_name<'a, A, Ix1> as IntoIterator>::Item;
+        type IntoIter = <$view_name<'a, A, Ix1> as IntoIterator>::IntoIter;
+
+        fn into_iter(self) -> Self::IntoIter {
+            self.0.into_iter()
+        }
```

Comment:
It actually reduces to the default iterator as a base case, that could be a regression in performance from currently (default iterator uses logical order)

Reply:
Visit order doesn't matter for 1D, only implementation quality (unless we count preferring forward over reverse, that is positive vs negative stride, and we don't have evidence for that).

Before we had the view's `self.into_iter().fold(..)`, so the fold case for the reduction. In the new indexed case, we don't always access the iterator using fold, and that's the possible quality problem.